index.js

```diff
@@ -63,11 +63,12 @@ async function handleRelease (robot, context) {
     .compact()
     .uniqBy('number')
     .forEach(async pullRequest => {
-      context.log(`PR: ${(await pullRequest).number}`)
+      const pr = await pullRequest
+      context.log(`PR: ${pr.number}`)
```

Comment:
we can remove line 66 altogether and just do `pullRequest.number`

Reply:
ok cool. That's kinda what I thought, but then I though there was something tricky about using async code in a forEach loop...? Anyway I changed it. Gonna ship it.